core-services/prow/02_config/shiftstack/prune/_pluginconfig.yaml

```diff
@@ -0,0 +1,45 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - shiftstack/prune
+  require_self_approval: false
+external_plugins:
+  shiftstack/prune:
+  - endpoint: http://refresh
+    events:
+    - issue_comment
+    name: refresh
+  - endpoint: http://needs-rebase
+    events:
+    - issue_comment
+    - pull_request
+    name: needs-rebase
+lgtm:
+- repos:
+  - shiftstack/prune
+  review_acts_as_lgtm: true
+plugins:
+  shiftstack/prune:
+    plugins:
+    - approve
+    - assign
+    - blunderbuss
+    - cat
+    - dog
+    - heart
+    - golint
+    - goose
+    - help
+    - hold
```

Comment:
Don't we need the jira plugin in case we want to link bugs to patches?

Reply:
:shrug: ok